_sass/base/components/_forms.scss

```diff
@@ -211,3 +215,23 @@ input[type=number]::-webkit-outer-spin-button {
 input[type=number] {
   -moz-appearance: textfield;
 }
+
+label.has-validity {
+  position: relative;
+
+  [aria-describedby] + span[id] {
+    display: none;
+  }
+
+  [aria-describedby]:invalid + span[id] {
+    display: block;
```

Comment:
Properties should be ordered background, display, left, margin-top, padding, position, top, width, z-index